server/sonar-server/src/test/java/org/sonar/server/user/UserUpdaterCreateTest.java

```diff
@@ -0,0 +1,557 @@
+/*
```

Comment:
remark: outside the scope of the PR, but I see several test classes for one production class as a sign to separate responsabilities